app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+import requests
+
+
+API_KEY = os.getenv("WEATHER_API_KEY")
+BASE_URL = "http://api.weatherapi.com/v1"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    payload = {
+        "key": API_KEY,
+        "q": FILTERING,
+    }
+    res = requests.get(BASE_URL + "/current.json", params=payload)
+    data = res.json()
+    location = data["location"]
+    current = data["current"]
```

Comment:
Accessing dictionary keys directly without checking if they exist can lead to `KeyError`. Consider using the `.get()` method or adding error handling to manage cases where the expected keys are not present in the response.